--- a/Imakefile
+++ b/Imakefile
@@ -15,7 +15,7 @@
 XCOMM  Set Has_GLX_SGI_video_sync to 0 if you get link errors like
 XCOMM  "ld: 0711-317 ERROR: Undefined symbol: .glXGetVideoSyncSGI"
 XCOMM  (e.g. under AIX 4.3)
-#define Has_GLX_SGI_video_sync 1
+#define Has_GLX_SGI_video_sync 0
 
 XCOMM  Set DoubleBuffer to 1 if your system has the include file
 XCOMM  <X11/extensions/Xdbe.h> and the library libXext.a
